web/src/utils/units.ts

```diff
@@ -0,0 +1,22 @@
+export enum PowerUnits {
+  WATTS = 'W',
+  KILOWATTS = 'kW',
+  MEGAWATTS = 'MW',
+  GIGAWATTS = 'GW',
+  TERAWATTS = 'TW',
+  PETAWATTS = 'PW',
+}
+
+export enum EnergyUnits {
+  WATT_HOURS = 'Wh',
+  KILOWATT_HOURS = 'kWh',
+  MEGAWATT_HOURS = 'MWh',
+  GIGAWATT_HOURS = 'GWh',
+  TERAWATT_HOURS = 'TWh',
+  PETAWATT_HOURS = 'PWh',
+}
+
+export enum CarbonUnits {
+  GRAMS_CO2EQ_PER_WATT_HOUR = 'gCO₂eq/kWh',
```

Comment:
If you want a space before and after the slash here let me know, this is how it was in the past but we use spaces elsewhere.